cds/modules/deposit/static/js/cds_deposit/avc/components/cdsUploader.js

```diff
@@ -139,6 +128,7 @@ function cdsUploaderCtrl(
               if (avcResponse) {
                 that.cdsDepositCtrl.presets = avcResponse.data.presets;
               }
+              that.cdsDepositsCtrl.fetchRecord();
```

Comment:
immediately fetch record after upload without waiting 15 seconds for the first fetch